EHR_App/test/src/org/labkey/test/tests/EHR_AppTest.java

```diff
@@ -0,0 +1,124 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.Connection;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.EHR;
+import org.labkey.test.tests.ehr.AbstractGenericEHRTest;
+import org.labkey.test.util.PostgresOnlyTest;
+
+import java.io.File;
+
+@Category({EHR.class})
+public class EHR_AppTest extends AbstractGenericEHRTest implements PostgresOnlyTest
+{
+    private static final String PROJECT_NAME = "EHR App";
+    private static final String FOLDER_NAME = "EHR";
+    private final EHRAppTestSetupHelper _setupHelper = new EHRAppTestSetupHelper(this, getProjectName(), FOLDER_NAME, getModulePath(), getContainerPath());
+
+    @Override
+    protected String getProjectName()
+    {
+        return PROJECT_NAME;
+    }
+
```

Comment:
```suggestion
    @Override
    protected boolean doSetUserPasswords()
    {
        return true;
    }
```

The built-in EHR permissions tests should pass if you add this function